src/main/java/seedu/duke/command/AddCommand.java

```diff
@@ -119,19 +123,27 @@ public void setDate(LocalDate date) {
      */
     @Override
     public void execute(TransactionList transactions, Ui ui, Storage storage) throws MoolahException {
-
+        statsLogger.setLevel(Level.WARNING);
+        statsLogger.log(Level.INFO, "Add Command checks the type of the transaction "
+                + "before adding into the transaction class.");
         assert date != null;
         switch (type) {
         case Expense.TRANSACTION_NAME:
             String expense = transactions.addExpense(description, amount, category, date);
             Ui.showTransactionAction(INFO_ADD_EXPENSE.toString(), expense);
+            statsLogger.log(Level.INFO, "New expense transaction has been added "
+                    + "and the UI should display respectively ");
             break;
         case Income.TRANSACTION_NAME:
             String income = transactions.addIncome(description, amount, category, date);
             Ui.showTransactionAction(INFO_ADD_INCOME.toString(), income);
+            statsLogger.log(Level.INFO, "New income transaction has been added "
+                    + "and the UI should display respectively ");
             break;
         default:
+            statsLogger.log(Level.INFO, "Exception thrown when the ransaction type is unknown");
```

Comment:
Maybe we can set the level for this as warning instead? Since it's an exception and this exception takes place at a rather unexpected place (by right when the command gets executed, the type must have been set to either income/expense already).

Reply:
Good point! Will update the level with the next commit.